### PR TITLE
[BEAM-4005] Futurize tools subpackage

### PR DESCRIPTION
**Pull request by Fematich**
This pull request prepares the tools subpackage for Python 3 support. This PR is part of a series in which all subpackages will be updated using the same approach.
This approach has been documented [here](https://s.apache.org/beam-python-3) and the first pull request in the series (Futurize coders subpackage) demonstrating this approach can be found at #5053.

R: @aaltay @tvalentyn

### CODE REVIEW

File: sdks/python/apache_beam/tools/distribution_counter_microbenchmark.py [hunk 1]
```diff
@@ -23,11 +23,16 @@
...
+from __future__ import absolute_import
+from __future__ import division
 from __future__ import print_function
 
 import random
 import sys
 import time
+from builtins import range
+
+from past.utils import old_div
```
Comment (RobbeSneyders) [inline sdks/python/apache_beam/tools/distribution_counter_microbenchmark.py]: Can you replace old_div with the new Python 3 division? Thanks.

Comment (Fematich) [reply sdks/python/apache_beam/tools/distribution_counter_microbenchmark.py]: Fixed in new commit

**Comment by Fematich** [comment]
run python precommit

**Comment by aaltay** [comment]
LGTM. Thank you.

**Pull request closed without merging**